### PR TITLE
Change the file size error to dynamically updated based on the consumer provided max file size.

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**
#48

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-file-size-dynamic-error](https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-file-size-dynamic-error)

File: src/app/public/modules/avatar/fixtures/avatar.component.fixture.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <sky-avatar
   [name]="name"
+  [maxFileSize]="maxFileSize"
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/avatar/fixtures/avatar.component.fixture.html:2]: Alphabetical ordering.

**Review by Blackbaud-SteveBrush — COMMENTED** [review]
Everything else looks good.

File: src/app/public/modules/avatar/fixtures/avatar.component.fixture.html (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/avatar/fixtures/avatar.component.fixture.html:2]: Fixed

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-file-size-dynamic-error](https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-file-size-dynamic-error)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-file-size-dynamic-error](https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-file-size-dynamic-error)

**Pull request merged**